### PR TITLE
Create models

### PR DESCRIPTION
**Pull request by vojtko84**
Create models

### CODE REVIEW

File: EntityFramework/Delivery/Delivery/Models/Deliveryman.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Delivery.Models
+{
+    public class Deliveryman : BaseModel
```
Comment (yevhenbilushko) [inline EntityFramework/Delivery/Delivery/Models/Deliveryman.cs:3]: DeliveryMan

File: EntityFramework/Delivery/Delivery/Models/Order.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        public DateTime OrderReceiptTime { get; set; }
+        public DateTime CustomerTransferTime { get; set; }
+        public int DeliverymanId { get; set; }
+        public int DeliveryRateId { get; set; }
+        public IList<Product> Products { get; set; }
+
+        public Order()
+        {
+            Products = new List<Product>();
+        }
```
Comment (yevhenbilushko) [inline EntityFramework/Delivery/Delivery/Models/Order.cs:20]: it is better not to have methods in EF model

**Comment by yevhenbilushko** [comment]
seems ok to me

File: EntityFramework/Delivery/Delivery/Models/Order.cs (hunk 2 above)
Comment (vojtko84) [reply EntityFramework/Delivery/Delivery/Models/Order.cs:20]: It is constructor. Or is it better not to have a constructor in the EF model too?

Comment (yevhenbilushko) [reply EntityFramework/Delivery/Delivery/Models/Order.cs:20]: is it better not to have a constructor in the EF model
EF model it is like SQL model but in C# language
it is should not have behavior and logic
it only for data

Comment (vojtko84) [reply EntityFramework/Delivery/Delivery/Models/Order.cs:20]: Ok, thanks for the explanation.

**Review by yevhenbilushko — APPROVED** [review]

**Pull request merged**